openedx/core/djangoapps/programs/utils.py

```diff
@@ -20,8 +19,10 @@
     get_run_marketing_url,
 )
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.user_api.preferences.api import get_user_preferences
 from openedx.core.djangoapps.programs.models import ProgramsApiConfig
 from openedx.core.lib.edx_api_utils import get_edx_api_data
+
```

Comment:
Revert this. First-party imports should appear in a single group, ordered alphabetically.